fe/fe-core/src/main/java/com/starrocks/http/rest/transaction/TransactionWithChannelHandler.java

```diff
@@ -48,6 +49,12 @@ public ResultWrapper handle(BaseRequest request, BaseResponse response) throws U
         Long timeoutMillis = txnOperationParams.getTimeoutMillis();
         String label = txnOperationParams.getLabel();
         Channel channel = txnOperationParams.getChannel();
+        String warehouseName = txnOperationParams.getWarehouseName();
+        Warehouse warehouse = GlobalStateMgr.getCurrentState().getWarehouseMgr().getWarehouse(warehouseName);
```

Comment:
‌‌‌Within the implementation of the getWarehouse operation, checks are conducted. If a valid Warehouse is not obtained, an exception will be thrown directly. Therefore, we can omit the check here for whether it is null.